models/README.md

```diff
@@ -0,0 +1,23 @@
+# Models
+
+There are **four** main models in this repository
+
+1. Lexical Model - Uses lexical features like POS tags, Named entities etc., followed by a Conditional Random Field model
+2. Lexical Plus Model - In addition to the lexical features, we engineered more optimization specific features followed by a Conditional Random Field model
+3. Semantic Model - Semantic model is a RoBERTa-based deep learning model fine-tuned on our training data. You find the model [here](https://huggingface.co/skadio/ner4opt-roberta-v1) 
+4. (Our best) **Hybrid Model** - This model combines the above feature engineering and feature learning models to get best results on our current dataset. 
+Please refer to our [training section](https://github.com/skadio/Ner4Opt/training) to understand how our best model is trained. 
+Our RoBERTa based model trained as a part of our hybridization experiment can be found [here](https://huggingface.co/skadio/ner4opt-roberta-v2)
```

Comment:
It is not clear to the reader what's the difference between this one and the v1 above.